shelley/chain-and-ledger/shelley-spec-ledger-test/test/Test/Shelley/Spec/Ledger/Fees.hs

```diff
@@ -508,12 +509,19 @@ testEvaluateTransactionFee :: Assertion
 testEvaluateTransactionFee =
   API.evaluateTransactionFee @(ShelleyEra C_Crypto)
     pp
-    (txSimpleUTxO @C_Crypto)
+    txSimpleUTxONoWit
     1
-    @?= Coin 103
+    @?= minfee pp (txSimpleUTxO @C_Crypto)
```

Comment:
I don't think it is a good idea to switch to `@?= minfee ...` because then this unit test only checks in the end that `evaluateMinFee` indeed calls `minfee`. Hard coding the expected value is better for a unit test like that. 

If it was a property test it would make more sense, but not for a unit test.

Also,  I might be misunderstanding somehting, but `evaluateTransactionFee` adds a dummy witnesses, but `minfee`  is getting a different witness, namely `makeWitnessesVKey (hashAnnotated $ txbSimpleUTxO @c) [alicePay]`, so it seem a bit hacky to me.

Reply:
exactly, `evaluateTransactionFee` is really just hoping to get `minfee` correct without having all the key witnesses yet.